src/app/containers/SocialEmbed/common/translations.js

```diff
@@ -1,54 +1,60 @@
 import pathOr from 'ramda/src/pathOr';
 
-const createTranslations = ({ translations, index }) => ({
-  fallback: {
-    text: pathOr(
-      'Content is not available',
-      ['socialEmbed', 'fallback', 'text'],
-      translations,
-    ),
-    linkText: pathOr(
-      'View content on %provider_name%',
-      ['socialEmbed', 'fallback', 'linkText'],
-      translations,
-    ),
-    linkTextSuffixVisuallyHidden: pathOr(
-      ', external',
-      ['socialEmbed', 'fallback', 'linkTextSuffixVisuallyHidden'],
-      translations,
-    ),
-    warningText: pathOr(
-      null,
-      ['socialEmbed', 'fallback', 'warningText'],
-      translations,
-    ),
-  },
+const createTranslations = ({ translations, index }) => {
+  const skipLinkText = pathOr(
+    'Skip %provider_name% content',
+    ['socialEmbed', 'skipLink', 'text'],
+    translations,
+  ).concat(index > 0 ? `, ${index}` : '');
 
-  skipLink: {
-    text: `${pathOr(
-      'Skip %provider_name% content',
-      ['socialEmbed', 'skipLink', 'text'],
-      translations,
-    )}, ${index}`,
-    endTextVisuallyHidden: `${pathOr(
-      'End of %provider_name% content',
-      ['socialEmbed', 'skipLink', 'endTextVisuallyHidden'],
-      translations,
-    )}, ${index}`,
-  },
+  const endTextVisuallyHiddenText = pathOr(
+    'End of %provider_name% content',
+    ['socialEmbed', 'skipLink', 'endTextVisuallyHidden'],
+    translations,
+  ).concat(index > 0 ? `, ${index}` : '');
 
-  caption: {
-    textPrefixVisuallyHidden: pathOr(
-      'Video caption, ',
-      ['socialEmbed', 'caption', 'textPrefixVisuallyHidden'],
-      translations,
-    ),
-    text: pathOr(
-      'Warning: Third party content may contain adverts',
-      ['socialEmbed', 'caption', 'text'],
-      translations,
-    ),
-  },
-});
+  return {
+    fallback: {
+      text: pathOr(
+        'Content is not available',
+        ['socialEmbed', 'fallback', 'text'],
+        translations,
+      ),
+      linkText: pathOr(
+        'View content on %provider_name%',
+        ['socialEmbed', 'fallback', 'linkText'],
+        translations,
+      ),
+      linkTextSuffixVisuallyHidden: pathOr(
+        ', external',
+        ['socialEmbed', 'fallback', 'linkTextSuffixVisuallyHidden'],
+        translations,
+      ),
+      warningText: pathOr(
+        null,
+        ['socialEmbed', 'fallback', 'warningText'],
+        translations,
+      ),
+    },
+
+    skipLink: {
+      text: skipLinkText,
+      endTextVisuallyHidden: endTextVisuallyHiddenText,
```

Comment:
I'd change `endTextVisuallyHiddenText` to `skipLinkEndTextVisuallyHidden`

```suggestion
      endTextVisuallyHidden: skipLinkEndTextVisuallyHidden,
```